technical-reports/Performance-Testing.md

```diff
@@ -0,0 +1,29 @@
+# Performance Testing of TEAMMATES
+
+This report gives a detailed explanation of the profiling operations performed on TEAMMATES. It gives an outline of the
+problem and describes our proposed solution.
+
+* [Introduction](#Introduction)
+* [Problem](#Problem)
+* [A brief overview of the Proposed Solution](#Overview-of-Solution)
+* [Tools considered for Performance Testing](#Tools-considered-for-Performance-Testing)
+* [Reasons for using JMeter](#Reasons-for-using-JMeter)
+* [Design of the Workflow](#Design-of-the-workflow)
```

Comment:
`Current implementation of the solution` instead? It will be a bit broader and we can discuss the reasons behind it too, rather than just describing the workflow